src/mds/CDir.cc

```diff
@@ -2922,8 +2935,6 @@ void CDir::scrub_initialize()
     }
   }
   scrub_infop->directory_scrubbing = true;
-
-  assert(scrub_local()); // TODO: handle failure
```

Comment:
I believe there was actually a purpose to calling it both times — we want to make sure that we aren't skipping any children on setup (so we check the in-memory state), and make sure that we haven't broken anything while the scrub is proceeding (so we check it again at the end). Maybe I'm misremembering and it was just an accident — it's not very expensive though, is it?